docs/instrumentations/overview.mdx

```diff
@@ -12,16 +12,18 @@ import { LanguagesCard } from '/snippets/languages-card.mdx';
 Odigos provides automatic instrumentations for the following runtimes:
 
 <LanguagesCard
-  golangUrl="/instrumentations/golang/ebpf"
+  golangUrl="/instrumentations/golang"
   golangDescription="Versions 1.17 and above are supported"
-  javascriptUrl="/instrumentations/nodejs/native"
+  javascriptUrl="/instrumentations/nodejs"
   javascriptDescription="Versions 14 and above are supported"
-  pythonUrl="/instrumentations/python/native"
+  pythonUrl="/instrumentations/python"
   pythonDescription="Versions 3.8 and above are supported"
-  javaUrl="/instrumentations/java/native"
+  javaUrl="/instrumentations/java"
   javaDescription="Versions 8 and above are supported"
-  dotnetUrl="/instrumentations/dotnet/native"
+  dotnetUrl="/instrumentations/dotnet"
   dotnetDescription="Versions 4.6.2 and above are supported"
+  phpUrl="/instrumentations/php"
+  phpDescription="Versions 8.0 and above are supported"
```

Comment:
suggestion: if in X mounts, version 8.6 of php is released, we will not support it out of the box.

WDYT about also capping the upper bound? to communicate exactly what we support.

for other runtimes we instrument regardless of the version, as long as it's above the minimum support